applications/luci-app-ddns/po/templates/ddns.pot

```diff
@@ -1,5 +1,15 @@
 msgid ""
-msgstr "Content-Type: text/plain; charset=UTF-8"
+msgstr ""
```

Comment:
@hnyman
I've seen this.
For the future we should think about to generate a complete header from the automatic script running.
If you open a .po and sync with the template the header is written into .po file.
Same if you create a new .po file from template.
I think, if someone out there wants to create a new translation all base data for the final .po should be prepared.
OK for this pull, I will lose the data on next run of automatic script.
Christian